docs/usage.rst

```diff
@@ -269,10 +269,9 @@ controlled identically to NumPy arrays, using ``numpy.setprintoptions``:
   >>> import numpy as np
   >>> import unyt as u
   ...
-  >>> np.set_printoptions(precision=4)
-  >>> print([1.123456789]*u.km)
+  >>> with np.printoptions(precision=4):
+  ...     print([1.123456789]*u.km)
```

Comment:
the reason this fix is needed is that the repr of `np.set_printoptions`'s return value is not empty in numpy 2.1, which caused this test to fail *and* leak side effects, causing a lot *other* tests to fail too.